soh/soh/Enhancements/randomizer/static_data.cpp

```diff
@@ -6,7 +6,7 @@ namespace Rando {
 
 std::unordered_map<uint32_t, CustomMessage> StaticData::hintTypeNames = {
     {HINT_TYPE_HINT_KEY, CustomMessage("Message")},
-    {HINT_TYPE_AREA, CustomMessage("Area")},
+    {HINT_TYPE_REGION, CustomMessage("Region")},
```

Comment:
Like the hint type, this is either Area or Region group. Note that this is put into the spoiler log.